nixos/doc/manual/configuration/fhs.chapter.md

```diff
@@ -0,0 +1,199 @@
+
+For example, with the following configuration:
+
+```nix
+  programs.nix-ld.enable = true;
+```
+
+...packages from conda can be used as follows:
+
+```bash
+$ nix-shell -p micromamba
+$ micromamba env create -n some-env python pytorch
+$ eval "$(micromamba shell hook --shell=bash)"
+$ micromamba activate some-env
+$ python
+>>> import torch
+```
+
+Without `nix-ld` it wouldn't be possible to run the conda-installed python
+interpreter, because it wouldn't know where to find the dynamic linker:
+
+```bash
+$ micromamba activate some-env
+$ python
+bash: /home/nixos/micromamba/envs/some-env/bin/python: cannot execute: required file not found
+$ patchelf --print-interpreter $(which python)
+/lib64/ld-linux-x86-64.so.2
+```
+
+Similarly, native libraries installed from binaries may be used with `nix-ld`:
+
+```bash
+$ python -m venv ~/some-venv
+$ source ~/some-venv/bin/activate
+$ pip install zmq
+$ python
+>>> import zmq
+```
+
+Without `nix-ld` the above would have failed:
+
+```bash
+$ source ~/some-venv/bin/activate
+$ python
+>>> import zmq
+...
+ImportError: libstdc++.so.6: cannot open shared object file: No such file or directory
+```
+
+A more complex scenario may be handled using ad hoc Nix shells:
+
+```nix
+# shell.nix
+
+with import <nixpkgs> { };
+
+mkShell {
+    packages = [ micromamba ];
+    NIX_LD_LIBRARY_PATH = lib.makeSearchPath "lib" [
+      addOpenGLRunpath.driverLink # "/run/opengl-driver"
+      glib.out
+      libglvnd
+      stdenv.cc.cc.lib
+    ];
+}
+```
+
+```bash
+$ nix-shell shell.nix
+$ micromamba env create -n omnimotion python=3.8
+$ eval "$(micromamba shell hook --shell=bash)"
+$ micromamba activate omnimotion
+$ micromamba install ...
+$ pip install ...
+```
+
+## General missing paths issues {#sec-fhs-missing-paths}
+
+Software built for "FHS" environments might sometimes also access other
+executables by their global "FHS" paths, instead of relying on the `PATH`
+variable and accessing them by name. E.g. it is common for shell scripts to
+start with a shebang like `#!/bin/bash` instead of `#!/usr/bin/env bash`. NixOS
+does not, by default, deploy `/bin/bash`, but it does deploy `/usr/bin/env`
+which can find `bash` using the `PATH` environment variable.
+
+NixOS provides the `services.envfs.enable` option for mounting a virtual
+filesystem in `/bin` and `/usr/bin`, which would resolve attempts at opening
+files in global locations (like `/bin/bash`) based on the `PATH` variable of
+the calling process:
+
+```nix
+  services.envfs.enable = true;
+```
+
+## Simulating an "FHS" environment {#sec-fhs-containers}
+
+Nixpkgs also provide tools for composing packages into a merged file tree. It
+is also possible to present this merged tree to "FHS" programs as overlaid on
+top of the real file system using the kernel feature called "user namespaces".
+This can be used to run "FHS" programs on NixOS Cf. [the Nixpkgs
+manual](https://nixos.org/manual/nixpkgs/unstable/#sec-fhs-environments)
```

Comment:
```suggestion
This can be used to run "FHS" programs on NixOS. Cf. [the Nixpkgs
manual](https://nixos.org/manual/nixpkgs/unstable/#sec-fhs-environments).
```